src/components/ShadowedCard/ShadowedCard.js

```diff
@@ -158,6 +160,7 @@ class ShadowedCard extends React.Component<Props, State> {
             style={upperContentWrapperStyle}
             isAnimated={isAnimated}
             opacity={contentOpacity}
+            borderRadius={borderRadius}
```

Comment:
Can we fallback to borderRadius of 6px here rather than in styled component declaration or change property name? It now overwrites border radius and sets it to 0 if borderRadius is not passed (check settings cards, for example).